planning/planning_test_utils/include/planning_interface_test_manager/planning_interface_test_manager_utils.hpp

```diff
@@ -235,10 +235,10 @@ HADMapBin makeMapBinMsg()
   return map_bin_msg;
 }
 
-Odometry makeOdometry()
+Odometry makeOdometry(const double shift = 0.0)
 {
   Odometry odometry;
-  const std::array<double, 4> start_pose{5.5, 4., M_PI_2};
+  const std::array<double, 4> start_pose{0.0, shift, 0.0, 0.0};
```

Comment:
@kyoichi-sugahara Why the pose was defined as follows?
```
const std::array<double, 4> start_pose{5.5, 4., M_PI_2};
```

Reply:
@TakaHoribe 
I copied from [here](https://github.com/autowarefoundation/autoware.universe/blob/c0ebaf351103c8343021b3d1f7e013b74462cfae/planning/freespace_planning_algorithms/test/src/test_freespace_planning_algorithms.cpp/#L42) to do similar test.
But in the interface test for freespace_planer, pose doesn't need to be defined with `const std::array<double, 4> start_pose{5.5, 4., M_PI_2};` and agree with your difinition.

Reply:
Thanks! done.